From 83570d26aeca95a5d81978c3ceca8f6a7d71596e Mon Sep 17 00:00:00 2001
From: Volker Krause <vkrause@kde.org>
Date: Sat, 8 Jul 2023 11:03:05 +0200
Subject: [PATCH] Fix binary stripping in APKs with Qt 5.15.10

This isn't as urgent as with the other apps due Tokodon's Android build
being broken anyway on binary factory at the moment. Once that is fixed
this change also needs to be backported to the release builds (in Craft).
---
 CMakeLists.txt       | 2 +-
 android/build.gradle | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b726ae0..ba32133 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,7 +14,7 @@ project(tokodon VERSION ${RELEASE_SERVICE_VERSION})
 set(KF_MIN_VERSION "5.77.0")
 set(QT_MIN_VERSION "5.15.0")
 if (ANDROID)
-    set(QT_MIN_VERSION "5.15.8")
+    set(QT_MIN_VERSION "5.15.10")
 endif()
 
 find_package(ECM ${KF_MIN_VERSION} REQUIRED NO_MODULE)
diff --git a/android/build.gradle b/android/build.gradle
index 2b1edf3..0f19946 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -47,6 +47,7 @@ android {
     compileSdkVersion androidCompileSdkVersion.toInteger()
 
     buildToolsVersion androidBuildToolsVersion
+    ndkVersion androidNdkVersion
 
     sourceSets {
         main {
-- 
2.41.0

